src/styles/utils/_variables.scss

```
@@ -0,0 +1,10 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$week-days:
+  "monday" 0,
+  "tuesday" ($cell-size + $cell-gap),
```

you can calculate this one time instead calc each time
```suggestion
$cell-gap: 1px;
$day-margin: $cell-size + $cell-gap;
$week-days:
  "monday" 0,
  "tuesday" $day-margin,
```